.env

```diff
@@ -0,0 +1,4 @@
+API_PORT=":1323"
+REDIS_PORT=":6379"
+AWS_ACCESS_KEY_ID=[---ENTER AWS ACCESS KEY ID HERE ---]
+AWS_SECRET_ACCESS_KEY=[---ENTER AWS SECRET ACCESS KEY HERE ---]
```

Comment:
Good points on this. I kept the AWS keys out for this reason. The reason I committed it was following `dotenv` suggestions in that we could have just development keys in this file that's checked into Git. 

But I can certainly remove it if it makes more sense/is safer to just keep these out of here.